src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-utils.c

```diff
@@ -608,13 +620,37 @@ static void xclmgmt_reset_pci(struct xclmgmt_dev *lro)
 	xclmgmt_config_pci(lro);
 }
 
+static void xclmgmt_debug_dump_uuid(struct xclmgmt_dev *lro)
+{
+	int node = -1;
+	const void *uuid;
+
+	if (!lro->core.fdt_blob)
+		return;
+
+	node = xocl_fdt_get_next_prop_by_name(lro, lro->core.fdt_blob,
+		-1, PROP_INTERFACE_UUID, &uuid, NULL);
+	if (!uuid || node < 0)
+		mgmt_dbg(lro, "no uuid");
+
+	mgmt_dbg(lro, "interface_uuid:%s", (char *)uuid);
+
+	node = xocl_fdt_get_next_prop_by_name(lro, lro->core.fdt_blob,
+		-1, PROP_LOGIC_UUID, &uuid, NULL);
+	if (!uuid || node < 0)
+		mgmt_dbg(lro, "no logic uuid");
+
+	mgmt_dbg(lro, "logic_uuid:%s", (char *)uuid);
+
+}
+
 int xclmgmt_update_userpf_blob(struct xclmgmt_dev *lro)
 {
 	int len = 0;
 	int userpf_idx = 0;
 	int ret = 0;
-	struct FeatureRomHeader rom_header = {};
-	struct VmrStatus vmr_header = {};
+	struct FeatureRomHeader rom_header = { 0 };
+	struct VmrStatus vmr_header = { 0 };
```

Comment:
any issue with the original code?

Reply:
it is a GNU C extension, not standard C99 or future C standard. But given this is indeed linux with GUN C as compiler. I changed it back.